evaluator/src/main/kotlin/PackageRule.kt

```diff
@@ -70,6 +70,35 @@ open class PackageRule(
         licenseRules.forEach { it.evaluate() }
     }
 
+    /**
+     * A [RuleMatcher] that checks whether a severity of any vulnerability of the [package][pkg] using the specified
+     * [scoringSystem] is higher than [maxValue]. Tested using the [severityComparator]. Returns false if the severity
+     * is equal to [maxValue].
+     */
+    fun hasVulnerabilityWithSeverityOver(
+        maxValue: String,
+        scoringSystem: String,
+        severityComparator: Comparator<String>
```

Comment:
Would it make sense to make these parameters optional and name the function just `hasVulnerability`, so user can also easy check whether there's *any* known vulnerability, without taking the score into account?